### PR TITLE
Fix so the library can be used as gem.

### PR DESCRIPTION
**Pull request by vilyapilya**
Previously the gem could not be used when installed. I removed unnecessary file structure complications and redundant complexities  from the gemspec file which caused the errors. 

Removed the executables since we don't have any of them in the project

### CODE REVIEW

File: lib/logdna-rails.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+# !/usr/bin/env ruby
```
Comment (dchai76) [inline lib/logdna-rails.rb]: Is this a bad lint thing? There shouldn't be a space after `#`

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (dchai76) [inline package-lock.json]: Uh, there shouldn't be a `package-lock.json` file in a Rails repo

**Review by dchai76 — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -1 +1,4 @@
 .DS_Store
+*.gem
+Gemfile.lock
+.package-lock.json
```
Comment (dchai76) [inline .gitignore]: Mmm, I'd get rid of this. Yes it should be ignored, but there's no reason why it should exist to begin with.

File: lib/logdna-rails.rb [hunk 4]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env ruby
-# encoding: utf-8
+#encoding: utf-8
```
Comment (dchai76) [inline lib/logdna-rails.rb]: This *should* have a space after '#'

Comment (dchai76) [reply lib/logdna-rails.rb]: This comment still needs addressing. Should probably just revert changes in this file.

**Review by dchai76 — CHANGES_REQUESTED** [review]

File: lib/logdna-rails.rb (hunk 4 above)
Comment (vilyapilya) [reply lib/logdna-rails.rb]: I can't revert the change bc the name of the should be changed. I'll add the space for the first comment as well for consistency.

**Review by dchai76 — APPROVED** [review]

**Pull request merged**